pkg/receive/hashring.go

```diff
@@ -39,23 +36,6 @@ type Hashring interface {
 	GetN(tenant string, timeSeries *prompb.TimeSeries, n uint64) (string, error)
 }
 
-// hash returns a hash for the given tenant and time series.
-func hash(tenant string, ts *prompb.TimeSeries) uint64 {
-	// Sort labelset to ensure a stable hash.
-	sort.Slice(ts.Labels, func(i, j int) bool { return ts.Labels[i].Name < ts.Labels[j].Name })
-
-	b := make([]byte, 0, 1024)
-	b = append(b, []byte(tenant)...)
-	b = append(b, sep)
```

Comment:
The separator is not appended in the new function, is this on purpose?

Reply:
Good finding! I don't think they are needed, but let's keep parity.